### ISSUE TITLE
Nemesis thread fails and test continues without nemesis after an error of: "Coordinator node timed out waiting for replica nodes"

### ISSUE THREAD

**Issue by yarongilor** [scylladb/scylla-cluster-tests#6019]
## Issue description

- [ ] This issue is a regression.
- [ ] It is unknown if this issue is a regression.

Nemesis thread failed with:
```
Traceback (most recent call last):
File "/home/ubuntu/scylla-cluster-tests/sdcm/sct_events/decorators.py", line 26, in wrapper
return func(*args, **kwargs)
File "/home/ubuntu/scylla-cluster-tests/sdcm/nemesis.py", line 387, in run
self.disrupt()
File "/home/ubuntu/scylla-cluster-tests/sdcm/nemesis.py", line 5429, in disrupt
self.call_next_nemesis()
File "/home/ubuntu/scylla-cluster-tests/sdcm/nemesis.py", line 1817, in call_next_nemesis
self.execute_disrupt_method(disrupt_method=self.disruptions_list.pop())
File "/home/ubuntu/scylla-cluster-tests/sdcm/nemesis.py", line 1745, in execute_disrupt_method
disrupt_method()
File "/home/ubuntu/scylla-cluster-tests/sdcm/nemesis.py", line 4314, in wrapper
args[0].cluster.check_cluster_health()
File "/home/ubuntu/scylla-cluster-tests/sdcm/cluster.py", line 4199, in check_cluster_health
node.check_node_health()
File "/home/ubuntu/scylla-cluster-tests/sdcm/cluster.py", line 2642, in check_node_health
events = self.node_health_events()
File "/home/ubuntu/scylla-cluster-tests/sdcm/cluster.py", line 2607, in node_health_events
group0_members = self.get_group0_members()
File "/home/ubuntu/scylla-cluster-tests/sdcm/cluster.py", line 3093, in get_group0_members
result = self.run_cqlsh(
File "/home/ubuntu/scylla-cluster-tests/sdcm/cluster.py", line 2799, in run_cqlsh
cqlsh_out = self.remoter.run(cmd, timeout=timeout + 30, # we give 30 seconds to cqlsh timeout mechanism to work
File "/home/ubuntu/scylla-cluster-tests/sdcm/remote/remote_base.py", line 613, in run
result = _run()
File "/home/ubuntu/scylla-cluster-tests/sdcm/utils/decorators.py", line 67, in inner
return func(*args, **kwargs)
File "/home/ubuntu/scylla-cluster-tests/sdcm/remote/remote_base.py", line 604, in _run
return self._run_execute(cmd, timeout, ignore_status, verbose, new_session, watchers)
File "/home/ubuntu/scylla-cluster-tests/sdcm/remote/remote_base.py", line 537, in _run_execute
result = connection.run(**command_kwargs)
File "/home/ubuntu/scylla-cluster-tests/sdcm/remote/libssh2_client/__init__.py", line 620, in run
return self._complete_run(channel, exception, timeout_reached, timeout, result, warn, stdout, stderr)
File "/home/ubuntu/scylla-cluster-tests/sdcm/remote/libssh2_client/__init__.py", line 654, in _complete_run
raise UnexpectedExit(result)
sdcm.remote.libssh2_client.exceptions.UnexpectedExit: Encountered a bad command exit code!

Command: 'cqlsh --no-color -u cassandra -p \'cassandra\' --request-timeout=120 --connect-timeout=60 --ssl -e "select server_id, can_vote from system.raft_state where group_id = 61d79f40-da3b-11ed-b74f-420a43312969 and disposition = \'CURRENT\'" 10.4.2.74 9042'

Exit code: 1

Stdout:



Stderr:

Connection error: ('Unable to connect to any servers', {'10.4.2.74:9042': ReadTimeout('Error from server: code=1200 [Coordinator node timed out waiting for replica nodes\' responses] message="Operation timed out for system_schema.columns - received only 0 responses from 1 CL=ONE." info={\'received_responses\': 0, \'required_responses\': 1, \'consistency\': \'ONE\'}',)})


ThreadFailedEventERRORevents
```

## Impact

*Describe the impact this issue causes to the user.*

## How frequently does it reproduce?

*Describe the frequency with how this issue can be reproduced.*

## Installation details

Kernel Version: 5.19.0-1022-aws
Scylla version (or git commit hash): `2023.1.0~rc4-20230402.f90af81a13cc` with build-id `4db7799d283c7ca63c7639b04db34c301dbcfb66`

Cluster size: 4 nodes (i3.4xlarge)

Scylla Nodes used in this run:
- longevity-tls-1tb-7d-2023-1-db-node-e34c1ae7-7 (3.252.121.165 | 10.4.0.79) (shards: 14)
- longevity-tls-1tb-7d-2023-1-db-node-e34c1ae7-6 (34.247.181.141 | 10.4.2.56) (shards: 14)
- longevity-tls-1tb-7d-2023-1-db-node-e34c1ae7-5 (52.30.188.16 | 10.4.1.109) (shards: 14)
- longevity-tls-1tb-7d-2023-1-db-node-e34c1ae7-4 (52.213.3.139 | 10.4.2.74) (shards: 14)
- longevity-tls-1tb-7d-2023-1-db-node-e34c1ae7-3 (3.252.160.44 | 10.4.1.141) (shards: 14)
- longevity-tls-1tb-7d-2023-1-db-node-e34c1ae7-2 (54.220.238.61 | 10.4.2.35) (shards: 14)
- longevity-tls-1tb-7d-2023-1-db-node-e34c1ae7-1 (54.217.89.108 | 10.4.1.54) (shards: 14)

OS / Image: `ami-000f0d43cb22bd09e` (aws: eu-west-1)

Test: `ics-longevity-1tb-7days-test`
Test id: `e34c1ae7-5ca9-4526-862f-817260f586a7`
Test name: `enterprise-2023.1/SCT_Enterprise_Features/ICS/ics-longevity-1tb-7days-test`
Test config file(s):

- [longevity-1TB-5days-authorization-and-tls-ssl.yaml](https://github.com/scylladb/scylla-cluster-tests/blob/1ff0b19195b9c45c13f9f7416c726acc5b3624c3/test-cases/longevity/longevity-1TB-5days-authorization-and-tls-ssl.yaml)

<details>
<summary>
Logs and commands
</summary>

- Restore Monitor Stack command: `$ hydra investigate show-monitor e34c1ae7-5ca9-4526-862f-817260f586a7`
- Restore monitor on AWS instance using [Jenkins job](https://jenkins.scylladb.com/view/QA/job/QA-tools/job/hydra-show-monitor/parambuild/?test_id=e34c1ae7-5ca9-4526-862f-817260f586a7)
- Show all stored logs command: `$ hydra investigate show-logs e34c1ae7-5ca9-4526-862f-817260f586a7`

## Logs:
*No logs captured during this run.*

[Jenkins job URL](https://jenkins.scylladb.com/job/enterprise-2023.1/job/SCT_Enterprise_Features/job/ICS/job/ics-longevity-1tb-7days-test/3/)
</details>

**Comment by yarongilor** [comment]
Not sure if it is related, but nemeses stopped running after a specific nemesis failure
[argus](https://argus.scylladb.com/test/a5d05355-2613-480b-9455-e462e0e78f28/runs?additionalRuns[]=5117ea34-02ba-40dc-bae8-b566f00feae0)

**Comment by yarongilor** [comment]
Should be fixed by `d3bb29f56 - (HEAD, upstream/branch-2023.1) fix(raft): make 'get_group0_members' work correctly when raft disabled`
